src/components/EditableInput/EditableInput.cy.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { EditableInput } from '@components/EditableInput/EditableInput';
+
+const EDITABLE_CONTAINER = '[data-test-id=editable-node-container]';
+const EDITABLE_LINK_NAME = '[data-test-id=node-link-name]';
+const EDITABLE_NODE = '[data-test-id=node-editable]';
+
+describe('EditableInput', () => {
+    it('renders without error', () => {
```

Comment:
Here you can do a `beforeEach` and mount the component instead of in every test:

```
const EDITABLE_INPUT_NAME = 'Editable Input';
const EDITABLE_INPUT_ID = 'ID';

beforeEach('Mount component', () => {
    const onEditableSaveSpy = cy.spy().as('onEditableSaveSpy');
    
    cy.mount(
        <EditableInput
            name={EDITABLE_INPUT_NAME}
            targetItemId={EDITABLE_INPUT_ID}
            onEditableSave={onEditableSaveSpy}
        />,
    );
}
```

It's a bit cleaner maybe.

Reply:
Thx for this input, was not thinking about mounting it in the beforeEach